src/helpers/helpers.dom.ts

```diff
@@ -1,4 +1,6 @@
+import {ChartEvent} from '../types';
 import {INFINITY} from './helpers.math';
+import {ChartArea, Scale, type Chart} from '../../types';
```

Comment:
is the `type` here necesarry? Since you are already importing it from the types I would think TS already knows its a type

Reply:
I think it would work since a class can be used as a "type", yes